src/lib/core/say.c

```diff
@@ -51,16 +51,24 @@
 
 pid_t log_pid = 0;
 /**
- * It is calculated as MAX(level, on_log_level) where level - is log level
```

Comment:
Please rewrite the commit message so it will be simple and understandable.

Reply:
Thanks for the patch!
Please rewrite the the second commit message so it will be simple and understandable, and it will be good for me.